README.md

```diff
@@ -1,21 +1,17 @@
-gov.au UI-Kit
+GOV.AU UI-Kit
 =============
 
-> The UI-Kit for the Australian Government.
+GOV.AU UI-Kit is a CSS and JS framework for Australian Government websites.
```

Comment:
I keep the first line stylistic different. Since we don't have much in markdown this is something a lot of people do. I don't think we need to think about semantics here?... Maybe?